requirements/development.txt

```diff
@@ -1,5 +1,6 @@
 black==22.12.0
-flake8>=4.0.1
+flake8==4.0.1
 notebook==6.4.12
+pylint==2.17.0
```

Comment:
do we want to fix versions of these? I fixed Black version because yearly changing style conventions Black is adopting.

Reply:
I don't know, tbh. It just seems safer to pin when in doubt, but if pylint's behavior is stable, maybe it's better not to.

Reply:
we can keep it this way for now